README.md

```diff
@@ -49,6 +49,7 @@ Experiment with flagd in your browser using [the Killercoda tutorial](https://ki
     ```
 
     Or use docker:
+   (for Windows use WSL both for the file path and the docker commands. This flagd on docker command WILL NOT WORK on Windows due to the mixed filesystems [#8479](https://github.com/docker/for-win/issues/8479)!! See also [this medium article](https://levelup.gitconnected.com/docker-desktop-on-wsl2-the-problem-with-mixing-file-systems-a8b5dcd79b22) )
```

Comment:
I would reword this section to something like the below (italic text & in a new line), 

```
    _Note - In Windows, use WSL system for both the file location and Docker runtime. Mixed file systems does not 
   work and this is a limitation of Docker (https://github.com/docker/for-win/issues/8479)_
```

I have mixed feelings on linking to the medium article as this is a premium access only. Plus, rewording to highlight this comes from Docker rather than flagd :)

Reply:
agreed.